tests/unittests/tests-phydat/tests-phydat.c

```diff
@@ -204,14 +204,14 @@ static void test_unitstr__success(void)
     TEST_ASSERT_EQUAL_STRING("m", phydat_unit_to_str_verbose(UNIT_M));
     TEST_ASSERT_EQUAL_STRING("m^2", phydat_unit_to_str_verbose(UNIT_M2));
     TEST_ASSERT_EQUAL_STRING("m^3", phydat_unit_to_str_verbose(UNIT_M3));
-    TEST_ASSERT_EQUAL_STRING("g", phydat_unit_to_str_verbose(UNIT_G));
```

Comment:
The symbol Wikipedia, the [CGPM](https://www.bipm.org/en/committees/cg/cgpm/3-1901/resolution-2) and [The Brochure](https://www.bipm.org/documents/20126/41483022/SI-Brochure-9-EN.pdf/2d2b50bf-f2b4-9661-f402-5f9d66e4b507?version=1.11&t=1671101192839#page=45) use `g_n` (g subscript n) when not spelling it out.

Reply:
We can do Unicode, so `gₙ`?

Reply:
hmm looks like the Unicode is causing some trouble.
We already have m^3, so g_n would just be natural

Reply:
Too bad ... any pointers to where the trouble manifested so we could fix things?

Reply:
nvm just a [missed instance](https://github.com/RIOT-OS/RIOT/compare/229ece38e929bbdf14ac6372a1334acd1e55ddc5..90137d07e8def018c14be0abdad15a81591443e7)